pagination/cursor/operations.graphql

```diff
@@ -0,0 +1,25 @@
+query FirstResults { 
```

Comment:
This should be a single operation using variables, then it shows how an app developer can use a single operation for pagination.

Reply:
c.f. https://github.com/stepzen-dev/snippets/blob/main/rest/pagination/requests.graphql#L2-L16